recipe/meta.yaml

```
@@ -22,7 +23,7 @@ requirements:
   run:
     - python
     - typing-extensions >=4.4
-    - pydantic >=1.8,<3
+    - pydantic >=1.10.17,<3
```

```suggestion
    - pydantic >=1.8,<3,!=1.10.15,!=1.10.17
```
https://github.com/aws/serverless-application-model/blob/v1.97.0/requirements/base.txt#L7C9-L7C37